selftests/deployment/roles/avocado/tasks/rpm/examples.yml

```diff
@@ -4,3 +4,13 @@
     name:
       - python3-avocado-examples
     state: latest
+  when:
+    - method == 'copr'
+
+- name: Install Avocado examples using RPM
+  dnf:
+    name:
+      - python-avocado-examples
```

Comment:
I'm not sure why we have different package names on upstream (thus on COPR) and on the official Fedora packages... I believe this is not intentional.

Anyway, your change is correct, but it'd be nice to look into that too.